source/habitat_risk_assessment.rst

```diff
@@ -108,17 +145,68 @@ For Multiplicative Risk calculation, risk to habitat *j* caused by stressor *k*
 
    2)	Cumulative risk in the grid cell is >66% of the total possible cumulative risk. Total possible cumulative risk is based on both the maximum risk score for an individual habitat (or species)-stressor combination and the maximum number of stressors that can occupy a particular grid cell in the study area (see next paragraph). Maximum number of overlapping stressors = 3 if, in the entire study region, no more than 3 stressors (e.g., agriculture run-off, marine aquaculture and marine transportation) are likely to occur in a single grid cell. Total possible cumulative risk in this case would be 8.49 (based on the Euclidean approach; the maximum risk score for a single habitat (or species)-stressor combination X the maximum number of overlapping stressors = 2.83 x 3 = 8.49). This criterion addresses the issue that even when a single stressor is not particularly detrimental the cumulative effect of multiple stressors causes is high.
 
 Cells are classified as MED if they have individual stressor or cumulative risk scores between 33%-66% of the total possible cumulative risk score. Cells are classified as LOW risk if they have individual or cumulative risk scores of 0-33% of the total possible risk score for a single stressor or multiple stressors, respectively.
 
-The maximum number of overlapping stressors is determined by the model. It is the total number of stressors in the study area; however, it is unlikely that all stressors will ever realistically overlap in a single grid cell. The model examines overlap in stressors to get the highest number of overlapping stressors.
+More formally, this is classified for each habitat/stressor pair as:
+
+.. math:: L_{jkl} = \begin{Bmatrix}
+        0 & if & R_{jkl} = 0 \\
+        1 & if & 0 < R_{jkl} < (\frac{1}{3}m_{jkl}) \\
+        2 & if & (\frac{1}{3}m_{jkl}) <= R_{jkl} < (\frac{2}{3}m_{jkl}) \\
+        3 & if & R_{jkl} >= (\frac{2}{3}m_{jkl})
+        \end{Bmatrix}
+   :label: hra-pairwise-risk-classification
+
+Where:
+
+* :math:`L_{jkl}` is the high/medium/low risk calculation for habitat :math:`j`
+  due to stressor :math:`k` at location :math:`l`.
+* :math:`R_{jkl}` is the computed risk of stressor :math:`k` to habitat
+  :math:`j` at location :math:`l`.
+* :math:`m_{jkl}` is the maximum score to each habitat/stressor pair, which is
+  consistent across all habitat/stressor pairs. This is defined as
+
+   * :math:`m_{jkl} = (r_{max})^2` if multiplicative risk is used.
+   * :math:`m_{jkl} = \sqrt{2(r_{max}-1)^2}` if euclidean risk is used.
+
+* :math:`r_{max}` is the user-defined maximum score.
+
+
+**Step 4.** Each grid cell for each habitat or species is classified as LOW
+(1), MED (2) or HIGH (3) risk based on risk posed by the cumulative effects of
+multiple stressors.
```

Comment:
I might not be reading this right, but don't we have a Step 4. already? Is leftover from the previous version?

Reply:
We sure do.  I've updated this in 7828994 in an attempt to better express how classification goes by doing the following:

* Reworking the steps 4 and 5 into 4, 4a and 4b to better reflect the interrelation of the classifications.
* The equations now more correctly (I think) capture the `max` part of `4b`.
* The equations in step 4 subsection 1 and step 4 subsection 2 are also linked to the related equations in 4a and 4b, respectively.